pkg/dockerregistry/server/repositorymiddleware.go

```diff
@@ -186,7 +186,6 @@ func newRepositoryWithClient(
 			return nil, fmt.Errorf("%s variable must be set when running outside of Kubernetes cluster", DockerRegistryURLEnvVar)
 		}
 	}
-	context.GetLogger(ctx).Infof("Using %q as Docker Registry URL", registryAddr)
```

Comment:
Unless someone runs it with a less verbosive level. Why do we make the `debug` level default anyway? As the name suggests, it should be enabled when debugging.

IMHO this statement is useful - at least in case of debugging. Since there were reports about too verbosive output, I'd vote for making `info` the default log level instead of erasing useful statements completely.